src/rpc/blockchain.cpp

```diff
@@ -2240,15 +2241,35 @@ static UniValue getblockfilter(const JSONRPCRequest& request)
  */
 UniValue dumptxoutset(const JSONRPCRequest& request)
 {
+    using cb_t = std::function<std::string(const COutPoint&, const Coin&)>;
+    const std::vector<std::pair<std::string, cb_t>> ascii_types{
+        {"txid",         [](const COutPoint& k, const Coin& c) { return k.hash.GetHex(); }},
+        {"vout",         [](const COutPoint& k, const Coin& c) { return ToString(static_cast<int32_t>(k.n)); }},
+        {"value",        [](const COutPoint& k, const Coin& c) { return ToString(c.out.nValue); }},
+        {"coinbase",     [](const COutPoint& k, const Coin& c) { return ToString(c.fCoinBase); }},
+        {"height",       [](const COutPoint& k, const Coin& c) { return ToString(static_cast<uint32_t>(c.nHeight)); }},
+        {"scriptPubKey", [](const COutPoint& k, const Coin& c) { return HexStr(c.out.scriptPubKey.begin(), c.out.scriptPubKey.end()); }},
+        // add any other desired items here
+    };
+
+    std::vector<RPCArg> ascii_args;
+    std::transform(std::begin(ascii_types), std::end(ascii_types), std::back_inserter(ascii_args),
+            [](const std::pair<std::string, cb_t>& t) { return RPCArg{t.first, RPCArg::Type::STR, RPCArg::Optional::OMITTED, "Info to write for a given UTXO"}; });
```

Comment:
It'd be nice if these were more descriptive. It's unclear exactly what the serialization is for these argsr